termios/termios_test.go

```diff
@@ -152,7 +153,7 @@ func checktty(t *testing.T, err error) {
 	t.Helper()
 	// some ioctls fail against char devices if they do not
 	// support a particular feature
-	if (runtime.GOOS == "darwin" && err == unix.ENOTTY) || (runtime.GOOS == "linux" && err == unix.EINVAL) {
+	if ((runtime.GOOS == "darwin" || runtime.GOOS == "linux") && err == unix.ENOTTY) || (runtime.GOOS == "linux" && err == unix.EINVAL) {
```

Comment:
It might make sense to put the `(err == unix.EINVAL || err == unix.ENOTTY)` together? Or maybe this is getting complex enough, we want to split it up in a `switch`?

🤷‍♀️ I mean, the code should work fine either way…

Reply:
I don't feel strongly either way. Happy to change it however you'd like, just let me know.